src/Raven.Server/Web/System/BackupDatabaseHandler.cs

```diff
@@ -63,5 +65,105 @@ public Task GetPeriodicBackupStatus()
 
             return Task.CompletedTask;
         }
+
+        [RavenAction("/periodic-backup/timer/all", "GET", AuthorizationStatus.ValidUser)]
+        public async Task GetAllPeriodicBackupsTimers()
+        {
+            if (TryGetAllowedDbs(dbName: null, out var dbs, requireAdmin: false) == false)
+                return;
+
+            var first = true;
+            var count = 0;
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                WriteStartOfTimers(writer);
+
+                foreach ((var name, Task<DocumentDatabase> task) in ServerStore.DatabasesLandlord.DatabasesCache)
+                {
+                    if (task.Status != TaskStatus.RanToCompletion)
+                        continue;
+
+                    if (dbs != null && dbs.ContainsKey(name.Value) == false)
+                        continue;
+
+                    var database = await task;
+                    if (database.PeriodicBackupRunner.PeriodicBackups.Count == 0)
+                        continue;
+
+                    if (first == false)
+                        writer.WriteComma();
+
+                    first = false;
+                    WritePeriodicBackups(database, writer, context, out int c);
+                    count += c;
+                }
+
+                WriteEndOfTimers(writer, count);
+            }
+        }
+
+        internal static void WriteEndOfTimers(AbstractBlittableJsonTextWriter writer, int count)
+        {
+            writer.WriteEndArray();
+            writer.WriteComma();
+            writer.WritePropertyName("TimersCount");
```

Comment:
Can you write this first?

Reply:
no, I get the count after I write all the backups